### ISSUE TITLE
Sidebar menu chevron alignment

### ISSUE THREAD

**Issue by vgr2** [mjbeller/web2py-starter#5]
When using Firefox, the chevron (arrow?) icon for each major menu entry is shown a line below the text it corresponds to.

**Comment by mjbeller** [comment]
I'm using the AdminLTE frontend framework ...
https://adminlte.io/themes/AdminLTE/index2.html

I think there is a new version that you could apply and see if that helps.  Please let me know if you find the solution.

**Comment by vgr2** [comment]
Fixed the issue. It turned out that a span was missing.

I've set it up with a request to merge the changes.

Disclaimer - first time submitting a to a private repo.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

"Nothing in the world can take the place of persistence.

Talent will not; nothing is more common than unsuccessful men with talent.

Genius will not; unrewarded genius is almost a proverb.

Education will not; the world is full of educated derelicts.

Persistence and determination alone are omnipotent.

The slogan Press On! has solved and always will solve the problems of the
human race."

-----

"We do not need more intellectual power, we need more spiritual power. We
do not need more of the things that are seen, we need more of the things
that are unseen."

— Calvin Coolidge
<http://www.goodreads.com/author/quotes/101882.Calvin_Coolidge>
